### ISSUE TITLE
getting a error while setting multi tenancy

### ISSUE THREAD

**Issue by mohitPanwar1996** [cube-js/cube#8000]
Hi everyone I am using cube cloud to do POC for our use case but i am struck at setting up the multi tenancy in cube.js file i have configured
contextToAppId, contextToOrchestratorId, driverFactory, scheduledRefreshContexts and when i am trying to query the models in the playground its throwing an error database "undefined" does not exist please can anyone can guide me on this in the sql runner i am getting all the tables and data of the respective database that i am setting dynamically i am posting screenshots below.

<img width="789" alt="Screenshot 2024-03-26 at 4 41 14 PM" src="https://github.com/cube-js/cube/assets/154788387/ed3c4659-05a4-447c-a186-f0e6f9feb745">
<img width="431" alt="Screenshot 2024-03-26 at 4 41 42 PM" src="https://gith
<img width="777" alt="Screenshot 2024-03-26 at 4 42 12 PM" src="https://github.com/cube
<img width="909" alt="Screenshot 2024-03-26 at 4 42 00 PM" src="https://github.com/cube-js/cube/assets/154788387/29a88731-426e-4671-a558-6ac0ea2b46f6">
-js/cube/assets/154788387/ba8a55ce-22ed-4edf-a28c-6bb63d7a0962">
ub.com/cube-js/cube/assets/154788387/3050e2e5-85c5-49cd-bd8d-e61ddd9925d0">

**Comment by igorlukanin** [comment]
Hi @mohitPanwar1996 👋 

I've tried to reproduce exactly your setup on the latest Cube version (0.35.7) and it works exactly as it should, correctly picking up the databases based on the security context:

<img width="1475" alt="Screenshot 2024-04-04 at 16 06 15" src="https://github.com/cube-js/cube/assets/3852894/9023c84b-6a91-422c-993f-36f6fa48facd">

<img width="1475" alt="Screenshot 2024-04-04 at 16 06 01" src="https://github.com/cube-js/cube/assets/3852894/65b1c6fd-cf37-4c99-bbd2-61fa88dea072">

Could you please upgrade to the latest version and double-check?

**Comment by igorlukanin** [comment]
@mohitPanwar1996 Have you tried the latest version? Is this issue still relevant?

**Comment by mohitPanwar1996** [comment]
Yes, updated to the latest cube version but it is still persistent and
happening in the cube cloud only not in cube-core.

**Comment by igorlukanin** [comment]
Could you please email a link to your deployment in Cube Cloud to igor@cube.dev? With your permission, I can check the logs of the deployment.
